### PR TITLE
Linter & Github Actions

### PR DESCRIPTION
**Pull request by bh2smith**
- Mostly a bunch of import rearrangements
- Change the eslintrc file
- Introduce `.github/workflows/CI.yml` which runs `yarn build`, `lint` and `coverage` (failing if any one of them fails)

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+module.exports = {
```
Comment (schmanu) [inline .eslintrc.js:1]: Looks great!

File: .github/workflows/CI.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      - uses: actions/cache@v2
+        with:
+          path: ${{ steps.yarn-cache.outputs.dir }}
+          key: ${{ matrix.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ matrix.os }}-yarn-
+      - run: yarn --frozen-lockfile
+      - run: yarn build
+      - run: yarn lint
+      - run: yarn coverage
```
Comment (schmanu) [inline .github/workflows/CI.yml:31]: Works fine as a first CI automation step.
I think in the long run we should probably split lint, test and coverage or at least lint and coverage to multiple jobs and probably include workflow badges on the repo readme. 
But thats optional.

**Review by schmanu — APPROVED** [review]
Really nice to have CI on github!

Great job of figuring it out! Thanks alot :unicorn:

File: .github/workflows/CI.yml (hunk 2 above)
Comment (bh2smith) [reply .github/workflows/CI.yml:31]: I fully intend on having a coverage badge when we get closer to 💯 at the very least.

**Pull request merged**